agogosml_cli/cli/templates/cookiecutter.json

```diff
@@ -7,6 +7,9 @@
     "PROJECT_NAME_SLUG": "sample_agogosml_project",
     "AZURE_CONTAINER_REGISTRY": "agogoscr.azurecr.io",
     "SUBSCRIPTION_ID": "",
+    "AGOGOSML_CLI_VERSION": "0.1.0",
+    "AGOGOSML_CLI_VERSION_MAJOR": "1",
+    "AGOGOSML_CLI_VERSION_MINOR": "0",
```

Comment:
isn't it "major.minor.rev/build" --- so, major is really `0` -- isn't it?

```
    "AGOGOSML_CLI_VERSION": "0.1.0",
    "AGOGOSML_CLI_VERSION_MAJOR": "0",
    "AGOGOSML_CLI_VERSION_MINOR": "1",
```

Reply:
I'll fix but it doesn't matter for this json file. It's just used as the template for cookiecutter. Values get set at generate time.